### PR TITLE
data: BruceWangyq

### PR DESCRIPTION
**Pull request by BruceWangyq**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2793"></a>

### CODE REVIEW

File: data/BruceWangyq.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/brucewangdev" },
+    { "icon": "FaGithub", "url": "https://github.com/brucewangyq" },
+    { "icon": "FaYoutube", "url": "https://youtube.com/@brucewangdev" }
+  ],
+  "links": [
+    {
+      "name": "Business: DevRel Services",
+      "url": "http://eddiejaoude.io",
```
Comment (Panquesito7) [inline data/BruceWangyq.json:23]: You might want to update this link.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by BruceWangyq** [comment]
This should be fixed! Thank you for pointing that out!

**Review by md-abid-hussain — APPROVED** [review]
Looks good to me

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good to me! 👍

**Comment by eddiejaoude** [comment]
Great collaboration 👍

**Comment by BruceWangyq** [comment]
Thank you so much for your work, guys~ It looks awesome!

**Comment by eddiejaoude** [comment]
Don't forget you can also add more information to your profile like
- events
- testimonials
- milestones 
with more features coming soon

**Pull request merged**